Source/WebCore/html/shadow/attachmentElementShadow.css

```diff
@@ -59,11 +63,19 @@ img#attachment-icon,
 div#attachment-placeholder,
 div#attachment-progress {
     grid-area: 1 / 1;
-    width: var(--icon-size);
-    aspect-ratio: 1;
+    align-self: center;
+    justify-self: center;
+}
+
+div#attachment-placeholder,
```

Comment:
I doubt this is the behavior we want for the placeholder? @squelart can you check?

Reply:
Yes we want that so that the placeholder has exactly the same size as icons.

Reply:
The current patch makes it so the placeholder has 3/4th the size of the icons, not sure that's expected

Reply:
Yes, it's expected. The full square icon width is too wide (most icons are naturally 3/4, so we're aiming for that.)